src/clr-icons/clr-icons-element.ts

```diff
@@ -57,6 +57,10 @@ ClarityIconElement.prototype._setIconSize = function(size: string) {
   }
 };
 
+ClarityIconElement.prototype._setRoleAttr = function(role: string) {
```

Comment:
we don't need this one now too. It's not called anywhere now

Reply:
Right, I forgot about that one. WIll remove.